src/hooks/listHooks.js

```diff
@@ -288,11 +288,18 @@ export const useRenderItem = (query, navigation, options = {}) => {
           return item.component;
         }
 
+        const bottomDivider =
+          query === QUERY_TYPES.PUBLIC_JSON_FILE
```

Comment:
there shouldn't be an exception for public json files. we want all lists behave the same, right?!